scripts/update_README.md

```diff
@@ -0,0 +1,28 @@
+# Continuous Update Delivery for WSO2 Identity Server
+
+The update script is to be used by WSO2 subscription users such that products packs can be updated. If you do not have a WSO2 subscription, you can sign up for a [free trial](https://wso2.com/subscription/free-trial).
+
+### Prequisites
+* A WSO2 Subscription
+* [WUM](https://wso2.com/updates/wum) updated packs should be provided in the `/modules/apim_common/files/packs` directory
+
+`
+NOTE: Provided WUM updated packs should contain the latest updates for wso2am-2.6.0 and wso2am-analytics-2.6.0
+`
+
+### Usage
+While executing the update script, provide the profile name. The pack corresponding to the profile will begin updating.
+```bash
+./update.sh -p <profile-name>
+```
+Any of the following profile names can be provided as arguments:
+* apim
+* apim_gateway
+* apim_km
+* apim_publisher
+* apim_store
+* apim_tm
+* apim_analytics_worker
+* apim_analytics_dashboard
```

Comment:
why do we have this?

Reply:
Fixed https://github.com/wso2/puppet-apim/pull/122/commits/9e657630dfe2bbf8bfbe082ace70eecb6f388ac4